### PR TITLE
apr dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-april-2023.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+Author: Kory Draughn
+Slug: irods-development-update-april-2023
+Status: published
+
+
+Happy Thursday Everyone,
+
+A lot has happened since last month. We are closer than ever to a release of iRODS 4.2.12. Our annual [iRODS User Group Meeting (UGM)](https://irods.org/ugm2023) is right around the corner as well. Registration is open for people who want to learn more about iRODS.
+
+For the server, we've made it so that `iexit` no longer removes the authentication file for the service account. This change protects administrators from losing the ability to manage their server. Of course, if you really need to remove the authentication file, then you have to option to forcibly remove it. `imkdir` has been tweaked to improve the performance in regard to inheritance and large collections. We've also implemented new microservices which give users the ability to inspect JSON objects using the iRODS Rule Language. This is specifically useful for handling PEPs that deal with JSON strings.
```
[the remark this answers is not in the document]
Comment (trel) [reply content/posts/irods-development-update-april-2023.md:12]: "improve the performance" -> "improve performance"

File: content/posts/irods-development-update-april-2023.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+We've fixed some important issues in the Logical Quotas rule engine plugin. It now uses the local rodsadmin user to update quota values. This avoids potential permission issues. Not only that, but it no longer crashes the agent when given unsupported rule text via `irule`.
+
+Development on the new iRODS CLI has started again. We don't have any specifics at this time, but as soon as we have something to show, you'll know about it.
+
+The Testing Environment has received a nice improvement in regard to topology testing. Topology tests can now be run in parallel. Before this change, the tests took upwards of 8 hours to complete. With this latest change, we've witnessed the tests taking as little as 2.25 hours to complete.
```
Comment (trel) [inline content/posts/irods-development-update-april-2023.md:18]: "2.25 hours to complete" -> "2.25 hours to complete on our development machines"

File: content/posts/irods-development-update-april-2023.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+    - develop tools to present iRODS as S3-compatible storage to existing S3 clients
+    - planning and investigation into custom C++ S3 implementation
+    - working meeting
+        - Fri, May 5, 2023, 3pm ET
+
+- [TRiRODS](https://irods.org/trirods)
+
+    - Wed, May 10, 2023, 4pm ET
+    - TBD
```
Comment (trel) [inline content/posts/irods-development-update-april-2023.md:66]: the roadmap?

Comment (korydraughn) [reply content/posts/irods-development-update-april-2023.md:66]: That wasn't included in the pc/twg notes.

File: content/posts/irods-development-update-april-2023.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (korydraughn) [reply content/posts/irods-development-update-april-2023.md]: Done.

File: content/posts/irods-development-update-april-2023.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-april-2023.md]: Done.

**Comment by korydraughn** [comment]
Squashed.

Please double check that I've made the appropriate changes correctly.

**Pull request merged**